packages/flutter_tools/templates/app_shared/linux.tmpl/CMakeLists.txt.tmpl

```diff
@@ -127,6 +127,14 @@ foreach(bundled_library ${PLUGIN_BUNDLED_LIBRARIES})
     COMPONENT Runtime)
 endforeach(bundled_library)
 
+# Copy the native assets provided by the build.dart from all packages.
+# For more info about native assets see:
+# https://github.com/flutter/flutter/issues/129757
```

Comment:
Issues shouldn't be used to document features. If there isn't a real documentation page to point users to, let's just omit this second sentence.

Reply:
(And the same change should be made in the migrator.)